services/ui-src/src/components/export/ExportedEntityDetailsTable.tsx

```diff
@@ -0,0 +1,172 @@
+import { ReactElement, useContext } from "react";
+// components
+import { ReportContext, Table } from "components";
+// types, utils
+import {
+  Choice,
+  EntityShape,
+  FieldChoice,
+  FormField,
+  ReportShape,
+  FormLayoutElement,
+  ReportType,
+} from "types";
+// verbiage
+import verbiage from "verbiage/pages/mcpar/mcpar-export";
+import { ExportedEntityDetailsTableRow } from "./ExportedEntityDetailsTableRow";
+
+export const ExportedEntityDetailsTable = ({
+  fields,
+  entity,
+  ...props
+}: Props) => {
+  const { report } = useContext(ReportContext);
+  const { tableHeaders } = verbiage;
+
+  const entityType = "program";
+
+  const threeColumnHeaderItems = [
+    tableHeaders.number,
+    tableHeaders.indicator,
+    tableHeaders.response,
+  ];
+
+  // The hint text is hidden for MLR page 1 (Point of Contact)
+  const reportType = report?.reportType as ReportType;
+  const hideHintText = reportType === ReportType.MLR;
```

Comment:
This code isn't specific to page one. Is there something I'm missing from context that makes it apply only to page 1, or am I being misled by the comment?